css/style.css

```diff
@@ -0,0 +1,435 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
I would rather see the SCSS files, and have the CSS file gitignored. Makes it easier to review how you're writing the SASS.